components/script/dom/imagebitmap.rs

```diff
@@ -0,0 +1,76 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+// the following are the statements
+use crate::dom::bindings::cell::DomRefCell;
+use crate::dom::bindings::codegen::Bindings::CanvasGradientBinding;
+use crate::dom::bindings::codegen::Bindings::CanvasRenderingContext2DBinding::CanvasImageSource;
+use crate::dom::bindings::codegen::Bindings::ImageBitmapBinding;
+use crate::dom::bindings::codegen::Bindings::ImageBitmapBinding::ImageBitmapMethods;
+use crate::dom::bindings::error::{Error, Fallible};
+use crate::dom::bindings::root::DomRoot;
+use crate::dom::blob::Blob;
+use crate::dom::globalscope::GlobalScope;
+use crate::dom::imagedata::ImageData;
```

Comment:
I think this is unused and can be removed.

Reply:
I wanted to confirm, aren't DomRoot and GlobalScope required in this implementation?
Because we seem to be using at line 49.
`    pub fn new(global: &GlobalScope, width: u32, height: u32) -> Fallible<DomRoot<ImageBitmap>>`

Reply:
Each of these comments only applies to the last line in the snippet shown. Github isn't great about making this clear, unfortunately.

Reply:
okay got it.